huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go

```diff
@@ -550,11 +628,17 @@ resource "huaweicloud_cce_node_pool" "test" {
   scale_down_cooldown_time = 0
   priority                 = 0
   type                     = "vm"
-  pod_security_groups      = [
+
+  security_groups = [
     huaweicloud_networking_secgroup.test1.id,
     huaweicloud_networking_secgroup.test2.id
```

Comment:
it seems that `huaweicloud_networking_secgroup.test2` has no rules, does it works?

Reply:
it works because huaweicloud_networking_secgroup.test1 has allowed the required ports
adding huaweicloud_networking_secgroup.test2 is just to check the result of attaching to secgroups